packages/vite/src/node/config.ts

```diff
@@ -1099,7 +1100,7 @@ async function loadConfigFromBundledFile(
   // for cjs, we can register a custom loader via `_require.extensions`
   else {
     const extension = path.extname(fileName)
-    const realFileName = fs.realpathSync(fileName)
+    const realFileName = await fsp.realpath(fileName)
```

Comment:
Note: The fsp version of `realpath` uses `.native` by default, which would cause file read errors on Windows for network drives. But since we're not doing that here, this change should be safe.

Reply:
This caused #12923, the async version returns a path with uppercase volume letter `C:\...` whereas the earlier version returns it lowercase.